### PR TITLE
SPO release notes

### PR DESCRIPTION
**Pull request by sheriff-rh**
Version(s):
4.12

Issue:
Release notes stub for https://github.com/openshift/openshift-docs/pull/52698/
https://issues.redhat.com/browse/OSDOCS-4766

Link to docs preview:
[Security Profiles Operator](http://file.rdu.redhat.com/antaylor/spo-rns/release_notes/ocp-4-12-release-notes.html#ocp-4-12-spo)

### CODE REVIEW

**Comment by sheriff-rh** [comment]
@rhmdnd @jhrozek PTAL - this is the release notes stub announcing SPO. The xref to the overview page does not work at this time, but will when the SPO docs are merged. This PR is based only on the 4.12 branch, which is why it had to be written separately. 

Please let me know if we need to make any changes!

**Review by jhrozek — COMMENTED** [review]
LGTM, just a question, do you think it makes sense to say that SPO also allows to record the profiles or do you prefer to keep the relnote short and to the point, since all the information is available in the SPO specific docs?

**Review by rhmdnd — COMMENTED** [review]
Looks good to me.

**Comment by mkumku** [comment]
Hi Andrew, where do you say that SPO is only compatible with OCP 4.12 and up? We agreed it woudl be in the release notes and I don't see it here.

**Comment by sheriff-rh** [comment]
> LGTM, just a question, do you think it makes sense to say that SPO also allows to record the profiles or do you prefer to keep the relnote short and to the point, since all the information is available in the SPO specific docs?

I think I want to keep it short and to the point here - we want to direct the reader to our section of content since it is an asynchronous project.

**Comment by sheriff-rh** [comment]
> Hi Andrew, where do you say that SPO is only compatible with OCP 4.12 and up? We agreed it would be in the release notes and I don't see it here.

Thanks for reminding me @mkumku , I have amended the statement to say "Security Profiles Operator (SPO) is now available for OpenShift Container Platform 4.12 and later."

**Comment by sheriff-rh** [comment]
/label peer-review-needed

Please note the disclaimer at the top - a failed Travis check is expected. This PR will be merged post-GA of 4.12 as it is asynchronous content.

**Comment by mjpytlak** [comment]
Good morning @sheriff-rh looks like you have a bad xref here, which is causing the build failure. Looks like missing everything past `#`.

`xref:../security/security_profiles_operator/spo-overview.adoc#`

**Comment by mjpytlak** [comment]
@sheriff-rh sorry missed your note about the expected failure, but still wondering if the xref is still missing the anchor ID.

**Review by mjpytlak — COMMENTED** [review]
Peer review LGTM @sheriff-rh

**Comment by jhrozek** [comment]
> > LGTM, just a question, do you think it makes sense to say that SPO also allows to record the profiles or do you prefer to keep the relnote short and to the point, since all the information is available in the SPO specific docs?
> 
> I think I want to keep it short and to the point here - we want to direct the reader to our section of content since it is an asynchronous project.

Thank you for the explanation, I agree. LGTM!
(Just let me know if I should formally click the accept button on the PR or add `lgtm` - I'm still unsure about the process for doc PRs)

**Comment by sheriff-rh** [comment]
> (Just let me know if I should formally click the accept button on the PR or add lgtm - I'm still unsure about the process for doc PRs)

No formal requirement, just the LGTM text is all we need! We're all set.

File: release_notes/ocp-4-12-release-notes.adoc [hunk 1]
```diff
@@ -571,10 +571,17 @@ A new import value, `importMode`, has been added to the `importPolicy` parameter
...
-//=== Security and compliance
-//
-// This content will be added post-GA, as it is asynchronous content.
+[id="ocp-4-12-security"]
+=== Security and compliance
+
+[id="ocp-4-12-spo"]
+==== Security Profiles Operator
+
+Security Profiles Operator (SPO) is now available for {product-title} {product-version} and later.
```
Comment (skrthomas) [inline release_notes/ocp-4-12-release-notes.adoc:580]:
```suggestion
The Security Profiles Operator (SPO) is now available for {product-title} {product-version} and later.
```

File: release_notes/ocp-4-12-release-notes.adoc [hunk 2]
```diff
@@ -571,10 +571,17 @@ A new import value, `importMode`, has been added to the `importPolicy` parameter
...
+
+SPO provides a way to define secure computing (https://kubernetes.io/docs/tutorials/security/seccomp/[seccomp]) profiles and SELinux profiles as custom resources, synchronizing profiles to every node in a given namespace.
```
Comment (skrthomas) [inline release_notes/ocp-4-12-release-notes.adoc:582]:
```suggestion
The SPO provides a way to define secure computing (https://kubernetes.io/docs/tutorials/security/seccomp/[seccomp]) profiles and SELinux profiles as custom resources, synchronizing profiles to every node in a given namespace.
```

**Review by skrthomas — COMMENTED** [review]
Need to add articles otherwise LGTM. Will merge once you push the changes.

**Comment by sheriff-rh** [comment]
Articles added. Ready for release.

**Pull request merged**